web/templates/twig-source/incuna-surveys/fields/base/checkbox.html

```diff
@@ -1,8 +1,11 @@
-<div drf-form-field="to.fieldOptions" class="checkbox">
-    <div ng-repeat="choice in to.choices" class="checkable checkbox">
-        {% raw %}
-        <input class="checkbox-input" id="{{ to.autoId }}-{{ $index }}" type="checkbox" checklist-model="model[to.fieldSetId][options.key]" checklist-value="$index">
-        <label class="checkbox-label" for="{{ to.autoId }}-{{ $index }}" ng-bind="choice"></label>
-        {% endraw %}
+{% extends "./base.html" %}
+{% block feild %}
+    <div drf-form-field="to.fieldOptions" class="checkbox">
+        <div ng-repeat="choice in to.choices" class="checkable checkbox">
+            {% raw %}
+            <input class="checkbox-input" id="{{ to.autoId }}-{{ $index }}" type="checkbox" checklist-model="model[to.fieldSetId][options.key]" checklist-value="$index">
```

Comment:
I left the indent the same as before, not sure if we should indent for `{% raw %}` tags  .